modules/custom-status/custom-status.php

```diff
@@ -232,36 +153,126 @@ public function disable_custom_statuses_for_post_type( $post_type = null ) {
 		return false;
 	}
 
-	public function add_admin_menu() {
+	/**
+	 * Checks for the current post type
+	 *
+	 * @return string|null $post_type The post type we've found, or null if no post type
+	 */
+	public static function get_current_post_type(): ?string {
```

Comment:
We could probably move this to `HelperUtilities`. It's currently only used in this file (and could also be made `private` instead), but I think it's a useful enough function to live somewhere shared.

Reply:
Done